### DISCUSSION TITLE
OpenIPC OSD is different from wfb_ng+wfb_ng_osd on ubuntu?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#337] [Questions] [answered]
How to configure ubuntu OSD to output the same stlye as OpenIPC do?

![图片](https://github.com/svpcom/wfb-ng/assets/1296171/1938e941-7a76-4926-899b-8bf51d00d428)

**Comment by svpcom** [comment]
OpenIPC OSD is related to openipc project. I don't know

**Reply by lida2003** [reply]
Thanks, I got it.

vdec add osd layer to the frame.

- https://github.com/OpenIPC/silicon_research
- "916 root     vdec -p 5600 -d frame -c h264 -m 1024x768x60 --mavlink-port 14750 --osd --bg-r 30 --bg-g 0 --bg-b 30"

**Reply by svpcom** [reply, marked as answer]
venc/vdec use incompatible with RTP stream format - i.e. you unable to play it using gstreamer pipelines or wfb-ng-osd
